<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ViDi - Visor Dicom</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Otros/resources/ViDi.svg</normaloff>:/Otros/resources/ViDi.svg</iconset>
  </property>
  <property name="windowOpacity">
   <double>25.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>414</width>
     <height>489</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="gv_images">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="tb_files">
   <property name="windowTitle">
    <string>Toolbar: General</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_open_file"/>
   <addaction name="separator"/>
   <addaction name="act_tools"/>
   <addaction name="separator"/>
   <addaction name="act_settings"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <widget class="QToolBar" name="tb_edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Toolbar: VIew</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_show_info"/>
   <addaction name="act_zoom"/>
   <addaction name="separator"/>
   <addaction name="act_roi_auto"/>
   <addaction name="act_roi_pol"/>
   <addaction name="act_roi_circ"/>
   <addaction name="act_clone_rois"/>
   <addaction name="act_get_stats"/>
  </widget>
  <widget class="QToolBar" name="tb_tools">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_dosimetry"/>
   <addaction name="act_phantom"/>
   <addaction name="act_normi13"/>
  </widget>
  <action name="actExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/exit.svg</normaloff>:/Actions/pictures/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit program</string>
   </property>
  </action>
  <action name="act_zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/zoom.svg</normaloff>
     <activeon>:/Actions/pictures/change-slice.svg</activeon>:/Actions/pictures/zoom.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom image/Move Z</string>
   </property>
  </action>
  <action name="actWL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/WL.svg</normaloff>:/Actions/pictures/WL.svg</iconset>
   </property>
   <property name="text">
    <string>Window/Level</string>
   </property>
   <property name="toolTip">
    <string>Change Window/Level</string>
   </property>
  </action>
  <action name="act_change_slice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/change-slice.svg</normaloff>:/Actions/pictures/change-slice.svg</iconset>
   </property>
   <property name="text">
    <string>Change image</string>
   </property>
   <property name="toolTip">
    <string>Navigate between images</string>
   </property>
  </action>
  <action name="act_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/settings.svg</normaloff>:/Actions/pictures/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Program settings</string>
   </property>
  </action>
  <action name="act_hybrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/hbrid.svg</normaloff>:/Actions/pictures/hbrid.svg</iconset>
   </property>
   <property name="text">
    <string>Hybrid imagesx</string>
   </property>
   <property name="toolTip">
    <string>Overlays images for a hybrid modality</string>
   </property>
  </action>
  <action name="act_roi_pol">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/new-pol-roi.svg</normaloff>:/Actions/pictures/new-pol-roi.svg</iconset>
   </property>
   <property name="text">
    <string>Draw roi</string>
   </property>
   <property name="toolTip">
    <string>Draw polygonal roi</string>
   </property>
  </action>
  <action name="act_roi_circ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/new-circ-roi.svg</normaloff>:/Actions/pictures/new-circ-roi.svg</iconset>
   </property>
   <property name="text">
    <string>Draw circular roi</string>
   </property>
   <property name="toolTip">
    <string>Draw circular roi</string>
   </property>
  </action>
  <action name="act_roi_auto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/auto-roi.svg</normaloff>:/Actions/pictures/auto-roi.svg</iconset>
   </property>
   <property name="text">
    <string>Automatic roi</string>
   </property>
   <property name="toolTip">
    <string>Automatically defines a roi</string>
   </property>
  </action>
  <action name="act_get_stats">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/roi-stats.svg</normaloff>:/Actions/pictures/roi-stats.svg</iconset>
   </property>
   <property name="text">
    <string>Roi stats</string>
   </property>
   <property name="toolTip">
    <string>Get roi statistics</string>
   </property>
  </action>
  <action name="act_select">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/select.svg</normaloff>:/Actions/pictures/select.svg</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>selection tool</string>
   </property>
  </action>
  <action name="act_rotate_cw">
   <property name="text">
    <string>Rotate CW</string>
   </property>
   <property name="toolTip">
    <string>Rotate image(s) 90ยบ clockwise</string>
   </property>
  </action>
  <action name="act_rotate_ccw">
   <property name="text">
    <string>Rotate CCW</string>
   </property>
   <property name="toolTip">
    <string>Rotate image(s) 90ยบ counter clockwise</string>
   </property>
  </action>
  <action name="act_normi13">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/normi-13.svg</normaloff>:/Actions/pictures/normi-13.svg</iconset>
   </property>
   <property name="text">
    <string>Normi 13</string>
   </property>
   <property name="toolTip">
    <string>Automatic analysis of normi 13 phantom</string>
   </property>
  </action>
  <action name="act_tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/tools.svg</normaloff>:/Actions/pictures/tools.svg</iconset>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="act_open_file">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/open-file.svg</normaloff>:/Actions/pictures/open-file.svg</iconset>
   </property>
   <property name="text">
    <string>open image(s)</string>
   </property>
   <property name="toolTip">
    <string>Open a single or serie of image(s)</string>
   </property>
  </action>
  <action name="act_open_3D">
   <property name="icon">
    <iconset>
     <normaloff>:/Actions/pictures/Open-3D.svg</normaloff>:/Actions/pictures/Open-3D.svg</iconset>
   </property>
   <property name="text">
    <string>open 3D image</string>
   </property>
   <property name="toolTip">
    <string>open 3D image (in one or several files)</string>
   </property>
  </action>
  <action name="act_dosimetry">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/dosimetry.svg</normaloff>:/Actions/pictures/dosimetry.svg</iconset>
   </property>
   <property name="text">
    <string>Internal dosimetry</string>
   </property>
   <property name="toolTip">
    <string>Opens internal dosimetry tool</string>
   </property>
  </action>
  <action name="act_show_info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>show info</string>
   </property>
  </action>
  <action name="act_clone_rois">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/clone-rois.svg</normaloff>:/Actions/pictures/clone-rois.svg</iconset>
   </property>
   <property name="text">
    <string>clone roi(s)</string>
   </property>
   <property name="toolTip">
    <string>Clone selected roi(s) to other slices</string>
   </property>
  </action>
  <action name="act_phantom">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Actions/pictures/phantom.svg</normaloff>:/Actions/pictures/phantom.svg</iconset>
   </property>
   <property name="text">
    <string>MIRD-5 phantom</string>
   </property>
   <property name="toolTip">
    <string>SPECT/CT image of MIRD-5 phantom</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>act_tools</sender>
   <signal>toggled(bool)</signal>
   <receiver>tb_tools</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
